source/usage-examples/replaceOne.txt

```diff
@@ -0,0 +1,46 @@
+==================
+Replace a Document
+==================
+
+.. default-domain:: mongodb
+
+You can replace a single document using the `replaceOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#replaceOne>`_
+method. ``replaceOne()`` accepts a query document and a replacement
+document, and replaces the first document that matches the filter using
```

Comment:
Nitpick: 
I wonder if is less ambiguous: "first document... using the ... replacement document" or "first document... **with** the ... replacement document"?

Reply:
Agreed, changed to your suggested wording